HoneyBadgers.RestApi/Services/ReportService.cs

```diff
@@ -4,18 +4,24 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using HoneyBadgers.RestApi.Context;
+using Microsoft.AspNetCore.Mvc;
 
 namespace HoneyBadgers.RestApi.Services
 {
     public class ReportService : IReportService
     {
         private readonly IRepository<ReportGenreStats> _reportGenreStatsRepository;
         private readonly IRepository<GenreStats> _genreStatsRepository;
+        private readonly IRepository<UserActivity> _usersActivityRepository;
+        private readonly HbReportContext _dbContext;
 
-        public ReportService(IRepository<GenreStats> genreStatsRepository, IRepository<ReportGenreStats> reportGenreStatsRepository)
+        public ReportService(IRepository<GenreStats> genreStatsRepository, IRepository<ReportGenreStats> reportGenreStatsRepository, HbReportContext dbContext, IRepository<UserActivity> usersActivityRepository)
         {
             _genreStatsRepository = genreStatsRepository;
             _reportGenreStatsRepository = reportGenreStatsRepository;
+            _dbContext = dbContext;
```

Comment:
Nie powinno być w Repository? Tak żeby to Repository się łączyło z bazą i robiło działania na bazie danych a serwis mówił tylko "repository wez mi tam dodaj actywność, a jak to zrobić to Twoja sprawa"? :)

Reply:
Poprawione, _dbContext wywalony